src/skeletons.tsx

```diff
@@ -0,0 +1,200 @@
+// https://www.figma.com/file/w7E0mmB92eio0zHw7h9iS2/%5BREADY%5D-Skeletons-Specs?node-id=986%3A1161
+import * as React from 'react';
+import SkeletonBase from './skeleton-base';
+import {createUseStyles} from './jss';
+import Stack from './stack';
+import classNames from 'classnames';
+import {getPrefixedDataAttributes} from './utils/dom';
+
+import type {DataAttributes} from './utils/types';
+
+type AnimationProps = {
+    disableAnimation: boolean;
+};
+
+const transition = '1.5s linear';
+
+const useAnimation = createUseStyles(() => ({
+    animation: {
+        animation: ({disableAnimation}: AnimationProps) =>
+            disableAnimation ? '' : `$pulse ${transition} infinite`,
+    },
+    '@keyframes pulse': {
+        '0%': {
+            opacity: 1,
+        },
+
+        '50%': {
+            opacity: 0.5,
+        },
+
+        '100%': {
+            opacity: 1,
+        },
+    },
+}));
+
```

Comment:
Did you consider apart of providing the possibility via prop disabling the animation by default based on reduced motion preferences?

Reply:
I've considered it but it's still an open issue AFAIK! I'd say we can proceed with this solution, because the father using these elements can control it, according to the needs (disabling, or other needs)

Reply:
we've decided to get rid off the disable animation feature completely